cmd/kubeadm/app/cmd/init.go

```diff
@@ -476,6 +476,13 @@ func (i *Init) Run(out io.Writer) error {
 		return fmt.Errorf("error creating clusterinfo RBAC rules: %v", err)
 	}
 
+	glog.V(1).Infof("[init] creating Nodes RBAC rules")
+	for _, token := range tokens {
+		if err := clusterinfophase.CreateNodesRBACRules(client, token); err != nil {
```

Comment:
those new rules, are they only for the check or to keep them forever ? I'm not sure that they need to stay in the cluster after kubeadm finishes the work.